### PR TITLE
Fix rails-erd

### PR DESCRIPTION
**Pull request by arthurnn**
```
bundle exec rake erd filetype=png filename=erd orientation=vertical title="RubyGems.org domain model"
```
Is not working at the moment. 
This pr fixes that, updating the gem version and also the diagram.

However, Do we still want to use this gem, to generate this diagram?
@sferik what are your thoughts (https://github.com/rubygems/rubygems.org/commit/225d1110dc8b70336134553a4a309de9fbc67796)

cc @dwradcliffe

### CODE REVIEW

**Comment by sferik** [comment]
I think having the ERD is helpful for newcomers to the project to understand our data model at a glance.

In the `twitter` gem, I [switch from using a bitmap-based PNG to vector-based SVG for the ERD](https://github.com/sferik/twitter/commit/58646faab6c62017cbf478164b58e38e4167e896). This has many advantages, including a smaller file size and the ability to more easily track changes over time with `git`.

The other thing I added to the `twitter` gem was an [`erd` rake task](https://github.com/sferik/twitter/blob/6750ece8a9c44482c073958814810b6cf27e1d1a/Rakefile#L4-L8) that generates an ERD before each release. This ensures that the ERD is up-to-date. Maybe we should have a similar task that hooks in to the `db:migrate` task. What do you think?

**Comment by simi** [comment]
I think it will be great to show some data structure image in **generated** documentation, but not in manually updated one.

**Comment by dwradcliffe** [comment]
I think this is helpful. Let's put that command in a rake task that doesn't need params, and maybe link to the image from the readme?

**Comment by arthurnn** [comment]
will do!

**Comment by arthurnn** [comment]
@dwradcliffe @sferik done.
I added a `rake gen_erd` task

**Comment by sferik** [comment]
@arthurnn Could you hook this in the run after the `db:migrate` task, so it runs automatically whenever the data model changes?

**Comment by arthurnn** [comment]
@sferik done.

File: Rakefile [hunk 1]
```diff
@@ -3,3 +3,11 @@ Gemcutter::Application.load_tasks
...
 desc "Run weekly at 00:00 UTC"
 task :weekly_cron => %w[gemcutter:rubygems:update_download_counts]
+
+task :gen_erd do
+  `bundle exec rake erd filetype=svg filename=doc/erd orientation=vertical title="RubyGems.org domain model"`
+end
+
+Rake::Task['db:migrate'].enhance do
+  Rake::Task['gen_erd'].invoke
+end
```
Comment (sferik) [inline Rakefile]: :+1:

**Comment by sferik** [comment]
@arthurnn What do you think about @dwradcliffe’s suggestion to display the image in the `README`? I like that idea too.

**Comment by arthurnn** [comment]
@sferik for sure... thanks for reminding me, I missed that.

**Comment by simi** [comment]
I think it will be better in DEVELOPMENT.md or in COTRIBUTING.md.

**Comment by arthurnn** [comment]
Turns out , this was on https://github.com/rubygems/rubygems.org/blob/master/CONTRIBUTING.md already.. I just fixed the link in this PR to use the .svg file

**Comment by dwradcliffe** [comment]
Thanks!

**Pull request merged**